templates/broadcast-message.tpl

```diff
@@ -22,6 +22,18 @@
 		<td><textarea class="flat" cols="40" rows="6" name="message"></textarea></td>
 	</tr>
 	<tr>
+		<td class="label"><label>{$PALANG.broadcast_mailboxes_only}</label></td>
+		<td><input type="checkbox" name="mailboxes_only"/></td>
```

Comment:
Your PHP code expects the checkbox value to be "on", but I wouldn't rely on that ;-)

It's better to use ` type="checkbox" value='1'`